libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -55,6 +55,13 @@ extern const AP_HAL::HAL& hal;
 
 #define MPU_SAMPLE_SIZE 14
 #define MPU_FIFO_BUFFER_LEN 16
+#if defined(STM32H7) || defined(STM32F7)
```

Comment:
this is a nice idea, just please add a comment explaining why this is being done

Reply:
Comment added